spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskControllerTests.java

```diff
@@ -641,7 +641,7 @@ public void testLaunchWithArguments() throws Exception {
 		verify(this.taskLauncher, atLeast(1)).launch(argumentCaptor.capture());
 
 		AppDeploymentRequest request = argumentCaptor.getValue();
-		assertThat(request.getCommandlineArguments()).hasSize(9);
+		assertThat(request.getCommandlineArguments()).hasSize(4);
```

Comment:
Magic number?

Reply:
There were additional commandline arguments added when launching apps to tell the applicaition what prefixes and boot versions to use.  With this PR these are no longer sent, thus the reduction from 9 to 4.